packages/xo-web/src/common/intl/messages.js

```diff
@@ -194,6 +194,9 @@ const messages = {
   editUserProfile: 'Edit my settings {username}',
 
   // ----- Home view ------
+  allVms: 'All VMs',
+  backedUpVms: 'Backed up VMs',
+  notBackedUpVms: 'Not backed up VMs',
```

Comment:
@Fohdeesha Does it make sense in english?

Reply:
Yes it reads correct to me!

Reply:
Thanks!